src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -416,8 +415,15 @@ async def build_pex(
 
     # Include any additional arguments and input digests required by the requirements.
     requirements_digests = []
-    pex_lock_resolver_args = [*python_repos.pex_args]
-    pip_resolver_args = [*python_repos.pex_args, "--resolver-version", "pip-2020-resolver"]
+    pex_lock_resolver_args = [
+        "--no-manylinux",  # manylinux is ignored, so we hardcode to avoid unnecessary invalidation.
```

Comment:
We could alternatively use the default of `manylinux2014`. It shouldn't really matter. Any preference?